### ISSUE TITLE
sudo apt-get install subgraph-os / debootstrap Subgraph OS

### ISSUE THREAD

**Issue by adrelanos** [subgraph/subgraph-os-issues#239]
(As [requested](https://secure-os.org/pipermail/desktops/2016-May/000117.html) earlier, now creating a ticket here for tracking purposes.)

Please provide instructions on how to `sudo apt-get install subgraph-os` / `debootstrap` Subgraph OS.

**Comment by mckinney-subgraph** [comment]
It will be pretty straight-forward once our full repo is up. 

Before that, you could try to get it working yourself with mixed repositories. I've never attempted to do that so I couldn't really advise you. There is a tool called `multistrap` that seems geared towards this purpose. If you get something working, let us know. Otherwise, this will stay open until our repo is up and `debootstrap` works normally.

**Comment by adrelanos** [comment]
What about "sudo apt-get install subgraph-os" from multiple repositories?

**Comment by mckinney-subgraph** [comment]
Not implemented. Feel free to implement it if you need it.

**Comment by xSmurf** [comment]
This seems like the wrong approach, the proper Debian way to do this would be using tasks.

As @mckinney-subgraph pointed out though we don't currently need this, so unless someone "PRs", this will go `wontfix`.

**Comment by adrelanos** [comment]
@xSmurf :
> This seems like the wrong approach, the proper Debian way to do this would be using tasks.

Do you mean tasks as per Debian's [`tasksel`](https://manpages.debian.org/jessie/tasksel/tasksel.8.en.html)?

In that case...

    tasksel --task-packages gnome-desktop

Outputs `task-gnome-desktop`.

Debian tasks (`tasksel`?) just installs a meta package. ([`task-gnome-desktop`](packages.debian.org/task-gnome-desktop) is an example one.)

I believe the proper Debian way would be using a meta package.

**Comment by xSmurf** [comment]
Yes tasks. A combinations of tasks and metapackages is how blends are built. Not everything is a dependency. What exactly is the point you are making here?

**Comment by adrelanos** [comment]
This ticket is a feature request to provide a `subgraph-os` meta package (and/or task) that can be installed using `sudo apt-get install subgraph-os` on plain Debian to convert plain Debian into Subgraph OS. That would be awesome.

**Comment by xSmurf** [comment]
As mentioned, this isn't needed in our current workflow so it's unlikely we will be dedicating resources to maintaining this. All the necessary information is available should someone decide they'd like to do it.
